libs/admin-app/profile/src/lib/account-information/account-information.component.ts

```diff
@@ -150,13 +151,23 @@ export class AccountInformationComponent implements OnInit {
         }
         else
         {
-          this.showMessage("Password", "Please make sure that your passwords match" )
+          this.toastService.show('Please make sure you that your passwords match',{
```

Comment:
We should probably not specify if it's the password or the id that is incorrect for security reasons, to  keep the data 
anonymous